src/config/konnectors.json

```diff
@@ -14,6 +14,10 @@
       "password": {
         "type": "password"
       },
+      "pathName": {
```

Comment:
Should not we just update the `FolderPickerField` component instead of adding a new one in every konnector ? I mean, we can maybe just made the `FolderPickerField` a more complex component by adding the new text field to it. What do you think @Narkfr and @CPatchane ?

Reply:
Totally !

Reply:
Completely agree 👍

Reply:
After some R&D and testing (https://github.com/mozilla-services/react-jsonschema-form), I think it's better to keep pathName (folderName is a better name btw) exposed in the manifest, to keep the rule "one declaration, one field".

This will keep `FolderPickerField` simplier, and I think it's a good idea to keep our form component the more simple possible, since our forms are already unvalid, not totally accessible, and buggy. And undocumented. (and I'm grummfy on sunday evening)